src/components/createPostModal/style.css

```diff
@@ -12,3 +12,11 @@ textarea {
     border-radius: 8px;
     padding: 16px;
 }
+.error
+{
```

Comment:
Bit picky but get these brackets up at the end of the classname, for consistency across the project